doc/manual/packages.rst

```diff
@@ -438,6 +439,38 @@ This means that :func:`Pkg.add("FooBar") <Pkg.add>` won't work yet since it only
 Moreover, when they run :func:`Pkg.update`, they will get the latest version of ``FooBar`` that you've pushed to the repo.
 This is a good way to have people test out your packages as you work on them, before they're ready for an official release.
 
+.. _man-manual-publish:
+
+Publishing METADATA manually
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If :func:`Pkg.publish` fails you can follow there instructions to
+manually publish your package.
+
+By "forking" the main METADATA repository, you can create a
+personal copy (of METADATA.jl) under your GitHub account. Once
+that copy exists, you can push your local changes to your copy
+(just like any other GitHub project).
+
+
+1. go to  `<http://github.com/JuliaLang/METADATA.jl>`_ and create your own
```

Comment:
You could even link to `https://github.com/JuliaLang/METADATA.jl/fork` directly?